scripts/build.js

```diff
@@ -16,10 +17,20 @@ const pkg = require('../package.json');
 
 const dirRoot = join(__dirname, '..');
 
+const bundledBuilders = Object.keys(pkg.devDependencies)
+  .filter(d => d.startsWith('@now/'));
+
+function getDistTag(version) {
```

Comment:
Looks like you defined this function twice.

Can it live in one place to keep it DRY?

Reply:
I had to convert the build script to TypeScript, but ya, done in 6e6b0bb3.